src/report_error.mli

```diff
@@ -0,0 +1,14 @@
+(** Error reporting *)
+
+(** Captures the backtrace and report an error.
+
+    Because jbuilder doesn't stop at the first error, it might end up reporting the same
+    error twice about missing libraries for instance. To avoid this, we keep a cache of
+    reported errors and ignore errors that have already been reported.
+
+    We cache what is actually printed to the screen.
+*)
+val report : exn -> unit
+
+(**/**)
+val map_fname : (string -> string) ref
```

Comment:
Is this `map_fname` set anywhere? Also, maybe a high level comment the purpose?

Reply:
That something we use at Jane Street for the public release process. Basically during the process files and we use this function to map errors to the original files rather than the exported projects. I've been meaning to change the code to just read the output of `jbuilder external-lib-deps` and use a regexp to change filenames so that this can be removed, but I haven't done it yet

Reply:
so eventually this will disappear. It's kind of ugly...